README.md

```diff
@@ -5,26 +5,85 @@ on Microsoft Azure REST APIs.
+| [osimage](/management/osimage)                                | [Linux OS images](https://azure.microsoft.com/documentation/articles/virtual-machines-linux-classic-about-images/) - [Windows OS images](https://azure.microsoft.com/documentation/articles/virtual-machines-windows-classic-about-images/)                           |
+| [sql](/management/sql)                                        | [SQL](https://channel9.msdn.com/Series/Windows-Azure-Storage-SQL-Database-Tutorials)                                                                                                                                                                                  |
+| [storageservice](/management/storageservice)                  | [Storage Accounts](https://azure.microsoft.com/documentation/articles/storage-create-storage-account-classic-portal/)                                                                                                                                                 |
+| [virtualmachine](/management/virtualmachine)                  | [Windows Virtual Machines](https://azure.microsoft.com/documentation/articles/virtual-machines-windows-about/) - [Linux Virtual Machines](https://azure.microsoft.com/documentation/articles/virtual-machines-linux-about/)                                           |
+| [virtualmachinedisk](/management/virtualmachinedisk)          | [Virtual Machine Disks](https://azure.microsoft.com/documentation/articles/virtual-machines-linux-about-disks-vhds/)                                                                                                                                                  |
+| [virtualmachineimage](/management/virtualmachineimage)        | [Linux VM images](https://azure.microsoft.com/documentation/articles/virtual-machines-linux-classic-about-images/) - [Windows VM images](https://azure.microsoft.com/documentation/articles/virtual-machines-windows-classic-about-images/)                           |
+| [virtualnetwork](/management/virtualnetwork)                  | [Virtual Networks](https://azure.microsoft.com/documentation/articles/virtual-networks-create-vnet-classic-portal/)                                                                                                                                                   |
+| [vmutils](/management/vmutils)                                | Utilities for managing Virtual Machines                                                                                                                                                                                                                               |
+| [storage](/storage)                                           | [Blobs](https://msdn.microsoft.com/library/azure/dd135733.aspx) - [Queue](https://msdn.microsoft.com/library/azure/dd179363.aspx) - [Table](https://msdn.microsoft.com/library/azure/dd179423.aspx) - [File](https://msdn.microsoft.com/library/azure/dn167006.aspx)  |
```

Comment:
storage is not part of ASM or ARM. it's a thing on its own. please separate into a different section.